.github/CHANGELOG.md

```diff
@@ -9,6 +9,17 @@
 
 ### Improvements
 
+* A new state-vector class `StateVectorManaged` was added, enabling memory use to be bound to 
+  statevector lifetime.
+  [(136)](https://github.com/PennyLaneAI/pennylane-lightning/pull/136)
+
+* The repository now has a well-defined component hierarchy, allowing each indepedent unit to be 
+  compiled and linked separately.
+  [(136)](https://github.com/PennyLaneAI/pennylane-lightning/pull/136)
+
+* PennyLane-Lightning now has support for the adjoint Jacobian method of http://arxiv.org/abs/2009.02823.
+  [(136)](https://github.com/PennyLaneAI/pennylane-lightning/pull/136)
```

Comment:
For me this is a major new feature :rocket: :tada: We can probably move it up during release :grin: